### PR TITLE
Fix a null reference error

### PR DESCRIPTION
**Pull request by BrianGenisio**
There are some times when the node is not in the list anymore.  In this case, it fails to add an entry to the cache and retrys over and over again, littering the console with errors

Handle this case by checking to see if the node is in the list.  If not, but a stub in place

### CODE REVIEW

File: src/components/node-manager.js [hunk 1]
```diff
@@ -108,8 +108,11 @@ class NodeManager {
      */
     layoutPropsForId(id) {
         if (!this._cachedBoundingBoxesById[id]) {
-            this._cachedBoundingBoxesById[id] =
-                this._nodesById[id].getBoundingClientRect();
+            const node = this._nodesById[id];
```
Comment (kevinbarabash) [inline src/components/node-manager.js]: Is the `id` the DOM id of the node?  If so, maybe we should try querying for it and only if we can't find then return a new `DOMRect()`.

File: yarn.lock [hunk 2]
```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
 
 
+"@babel/runtime@^7.1.2":
```
Comment (kevinbarabash) [inline yarn.lock]: I assume these changed because this yarn.lock was created with a really old version of yarn.

Comment (BrianGenisio) [reply yarn.lock]: I assume.  I just ran `yarn install` and it updated.

File: src/components/node-manager.js (hunk 1 above)
Comment (BrianGenisio) [reply src/components/node-manager.js]: This is really difficult to replicate.  Requires me to run in a phone emulator, and a bunch of steps... so I'd rather not try to get too tricky here.

**Comment by BrianGenisio** [comment]
Thanks Kevin.  I can confirm that an empty DOMRect is all `0`s.

**Pull request merged**